src/planning/src/adapter_planning/pacsim.cpp

```diff
@@ -57,7 +57,8 @@ void PacSimAdapter::track_callback(const visualization_msgs::msg::MarkerArray& m
   RCLCPP_DEBUG(this->get_logger(), "Planning : Received track map in pacsim adapter");
   custom_interfaces::msg::ConeArray cones;
   for (auto c : msg.markers) {
-    if (c.type == 4 || (c.pose.position.x == 0 && c.pose.position.y == 0)) {
+    if (c.type == 4 || (c.pose.position.x == 0 && c.pose.position.y == 0) ||
```

Comment:
What is this for?

Reply:
Pacsim publishes the map with some markers that represent things other than the cones. That code's goal is to remove those markers